zc_common/remote_resource/test_mixins.py

```diff
@@ -386,7 +391,7 @@ def test_resource(self):
         self.assertIsNotNone(deleted_resource[0].deleted_at)
 
     def test_delete__404_for_nonexistant_resource(self):
-        url = reverse(self.resource_view_name, args=(self.resource.id + 10000,))
```

Comment:
How it was previously written works when the resource has an integer as its primary key, but if you have something with a non-integer such as a UUID, Python will throw an error because it's trying to perform addition on a str and an int. Since we're trying to test for a 404/nonexistent resource, if we replace the entire calculation with an id we know will not exist as an integer and pass it via args to reverse here, Django will correctly cast it to a string or an integer depending on the field type.